jupyterlab_hdf/util.py

```diff
@@ -101,6 +107,8 @@ def hobjType(hobj):
         return 'dataset'
     elif isinstance(hobj, h5py.Group):
         return 'group'
+    elif isinstance(hobj, HobjExternalLink):
+        return 'link'
```

Comment:
Added a new `link` type to handle things in the front-end.

Reply:
The thing is, there's already is a builtin [`h5py.ExternalLink`](https://docs.h5py.org/en/stable/high/group.html#h5py.ExternalLink) in `h5py`

Reply:
there should probably be both `'link'` and `'externalLink'`.

Possibly there should be all three of `'softLink'`, `'hardLink'`, and `'externalLink'`. But we shouldn't actually need to distinguish hard links and regular groups, so `'softLink'` can just be shortened to `'link'`